ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOtherDirectionHeaderType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
+
+public enum CCDOtherDirectionHeaderType {
+    HEADER_UPLOAD,
```

Comment:
HEADER word seems redundant. can you please remove the word HEADER in front of UPLOAD and CONFIRM ? Thanks

Reply:
I can't, because that's used in the CCD UI - in the def file we have something like `"otherDirectionHeaders=\"HEADER_*\""},`

Reply:
Correct me if I am wrong - In CCD UI, HEADER_* and * wont make difference, as all our types start with HEADER word. The condition should be if it has value show the `directionComment`. Also we cant store any value in header as it accept only fixed list values.

Reply:
@karthikkannankainos you're actually right, I think I can remove HEADER_ and leaving * . Not sure why I didn't see it before :D sorry! will change